Ecommerce/index.html

```diff
@@ -1,32 +1,76 @@
 
-<!DOCTYPE html>
-<html><head><title>Task1</title></head>
-<body>
-<h1>Welcome to My Shop</h1>
-<img src="shp.png">
-<ul>
-<li><a href="#">About Us</a></li>
-<li><a href="#">Products</a></li>
-<li><a href="#">Contact Us</a></li>
-</ul>
-<h2>About Us</h2>
-Welcome to My Shop! We are a one -stop destination for all your shopping needs. Our mission is to provide you with the best products and exceptional customer service.
-<h2>Products</h2>
-Explore our wide range of products, including:

```

Comment:
Consider specifying a fallback font in case the specified font is not available on the user's system for better cross-browser compatibility.